ntc_rosetta/translators/openconfig/ios/openconfig_interfaces/interfaces.py

```diff
@@ -76,6 +82,12 @@ def enabled(self, value: Optional[bool]) -> None:
         else:
             self.yy.result.add_command(f"   shutdown")
 
+    def mtu(self, value: Optional[int]) -> None:
+        if value:
+            self.yy.result.add_command(f"   mtu {value}")
+        else:
+            self.yy.result.add_command(f"   no mtu")
+
```

Comment:
As you are parsing it, I think it would be nice to also translate the loopback interface